mdata/init.go

```diff
@@ -15,6 +15,13 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	discardReasonLabel = "reason"
```

Comment:
seems weird to declare this as a global since it only has 1 use, and for org we don't do this.

Reply:
indeed! fixed.